pkg/evaluation/evaluation.go

```diff
@@ -31,6 +31,10 @@ import (
 	pkgdocumentloaders "github.com/kubeagi/arcadia/pkg/documentloaders"
 )
 
+const (
+	maxContentLength = 500 // Maximum length of context. Contents which exceed this length will be truncated.
```

Comment:
@Lanture1064 Here  we can make this as a genOptions https://github.com/kubeagi/arcadia/blob/main/pkg/evaluation/evaluation.go#L82 and give it `-1` which means no limit.

Reply:
After LLM add the above configuration,we can pass that argument as  in our `arctl eval` command.